api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -16,58 +16,73 @@ service Disperser {
   rpc GetBlobStatus(BlobStatusRequest) returns (BlobStatusReply) {}
-  
+
   // GetBlobCommitment is a utility method that calculates commitment for a blob payload.
   rpc GetBlobCommitment(BlobCommitmentRequest) returns (BlobCommitmentReply) {}
 
   // GetPaymentState is a utility method to get the payment state of a given account.
   rpc GetPaymentState(GetPaymentStateRequest) returns (GetPaymentStateReply) {}
 }
 
-// Requests and Replys
+// Requests and Replies
 
+// A request to disperse a blob.
 message DisperseBlobRequest {
   // The data to be dispersed.
-  // The size of data must be <= 16MiB. Every 32 bytes of data is interpreted as an integer in big endian format
-  // where the lower address has more significant bits. The integer must stay in the valid range to be interpreted
-  // as a field element on the bn254 curve. The valid range is
-  // 0 <= x < 21888242871839275222246405745257275088548364400416034343698204186575808495617
+  //
+  // Although the length commitment is required to be a power of 2, the length of this byte array is not required to
+  // be a power of 2 as long as the length does not exceed the length commitment. The maximum length of the data is
+  // 16MiB. (In the future, the 16MiB limit may be increased, but this is not guaranteed to happen.)
+  //
+  // Every 32 bytes of data is interpreted as an integer in big endian format where the lower address has more
+  // significant bits. The integer must stay in the valid range to be interpreted as a field element on the bn254 curve.
+  // The valid range is 0 <= x < 21888242871839275222246405745257275088548364400416034343698204186575808495617.
   // If any one of the 32 bytes elements is outside the range, the whole request is deemed as invalid, and rejected.
   bytes data = 1;
+  // The header contains metadata about the blob.
   common.v2.BlobHeader blob_header = 2;
 }
 
+// A reply to a DisperseBlob request.
 message DisperseBlobReply {
   // The status of the blob associated with the blob key.
   BlobStatus result = 1;
+  // The unique identifier for the blob. Unique even if blob data is the identical.
   bytes blob_key = 2;
 }
 
 // BlobStatusRequest is used to query the status of a blob.
 message BlobStatusRequest {
+  // The unique identifier for the blob.
   bytes blob_key = 1;
 }
 
+// BlobStatusReply is the reply to a BlobStatusRequest.
 message BlobStatusReply {
   // The status of the blob.
   BlobStatus status = 1;
   // The signed batch
   SignedBatch signed_batch = 2;
+  // BlobVerificationInfo is the information needed to verify the inclusion of a blob in a batch.
   BlobVerificationInfo blob_verification_info = 3;
 }
 
-// Utility method used to generate the commitment of blob given its data.
-// This can be used to construct BlobHeader.commitment
+// A request to generate the commitment of a blob via BlobCommitmentRequest().
+// This can be used to construct a BlobHeader.commitment.
 message BlobCommitmentRequest {
+  // The blob data.
```

Comment:
"The blob data to compute commitment for"?

Reply:
change made